Tasks/NuGetInstaller/task.loc.json

```diff
@@ -33,14 +33,6 @@
       "required": true
     },
     {
-      "name": "excludeVersion",
-      "type": "boolean",
-      "label": "ms-resource:loc.input.label.excludeVersion",
-      "defaultValue": "false",
-      "helpMarkDown": "ms-resource:loc.input.help.excludeVersion",
```

Comment:
We should remove these strings from the strings.json file. Not sure why the build process didn't generate a different file there.